google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingHandler.java

```diff
@@ -372,22 +373,9 @@ public void publish(LogRecord record) {
 
     try {
       LogEntry entry = entryFor(record);
-
-      List<LogEntry> flushBuffer = null;
-      WriteOption[] flushWriteOptions = null;
-
-      synchronized (this) {
-        if (entry != null) {
-          buffer.add(entry);
-        }
-        if (buffer.size() >= flushSize || record.getLevel().intValue() >= flushLevel.intValue()) {
-          flushBuffer = buffer;
-          flushWriteOptions = writeOptions;
-          buffer = new LinkedList<>();
-        }
+      if (entry != null) {
+        write(entry, writeOptions);
```

Comment:
We are losing the ability to trigger a flush if we encounter a log at the configured flush level. I realize that the existing implementation doesn't actually flush, but I also don't want to toss this feature away. Add a note to #1795 about this.